src/components/InventoryDetail/DetailWrapper.test.js

```diff
@@ -80,61 +85,68 @@ describe('DetailWrapper', () => {
   });
 
   describe('DOM', () => {
+    beforeAll(() => {
+      BasicInfo.mockImplementation(
+        jest.requireActual('../InventoryDetailDrawer').BasicInfo
+      );
+    });
+
     it('should render without data', () => {
       const store = mockStore({
         entityDetails: {},
       });
-      const wrapper = mount(
+      const view = render(
         <Provider store={store}>
           <DetailWrapper />
         </Provider>
       );
-      expect(
-        toJson(wrapper.find('DetailWrapper'), { mode: 'deep' })
-      ).toMatchSnapshot();
+
+      expect(view.asFragment()).toMatchSnapshot();
     });
 
     it('should render with data', () => {
       const store = mockStore(initialState);
-      const wrapper = mount(
+      const view = render(
         <Provider store={store}>
           <DetailWrapper />
         </Provider>
       );
-      expect(
-        toJson(wrapper.find('DetailWrapper'), { mode: 'deep' })
-      ).toMatchSnapshot();
+
+      expect(view.asFragment()).toMatchSnapshot();
     });
 
     it('should pass correct props to BasicInfo', () => {
+      BasicInfo.mockImplementation(() => <span>BasicInfo</span>);
       const store = mockStore(initialState);
-      const wrapper = mount(
+      render(
         <Provider store={store}>
           <DetailWrapper />
         </Provider>
       );
-      expect(
-        toJson(wrapper.find('BasicInfo'), { mode: 'shallow' })
-      ).toMatchSnapshot();
+
+      expect(BasicInfo).toHaveBeenCalledWith(
```

Comment:
This test is a bit odd. Maybe we can avoid the mocking of `BasicInfo` and let it just render as normal and query for the effects of the props? For example, query if tags are or are not shown, when passing in `showTags`.